upload.py

```diff
@@ -73,6 +73,41 @@ def write_to_report(props, text):
     props.report = props.report + " - " + text + "\n\n"
 
 
+def prevalidate_model(props):
+    """ "Check model for possible problems"""
+    ob = utils.get_active_model()
+    obs = utils.get_hierarchy(ob)
+    # check scaling of objects, if anything is assymetrical, it's a big problem,
+    # if anything is scaled away from (1,1,1), it's a smaller problem.
+    for ob in obs:
+        if ob.scale[0] != ob.scale[1] or ob.scale[1] != ob.scale[2]:
+            write_to_report(
+                props,
+                "Assymetrical scaling in the object - please apply scale on all models",
+            )
+
+
+def prevalidate_scene(props):
+    # check scene for possible problems
```

Comment:
Docstring has to be in format `"""Check scene for possible problems."""` Would be nice to describe the actual possible problems this function covers.